packages/web/components/Dashboard/Settings/Subscription/SubscriptionPlanSelect.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import {
+  MembershipSubscriptionPeriod,
+} from '@/generated/graphql'
+import Select from '@/components/Select'
+
+type SubscriptionPlanSelectProps = {
+  selectedOption: MembershipSubscriptionPeriod
+  setSelectedOption: (arg1: MembershipSubscriptionPeriod) => void
+}
+
+const SubscriptionPlanSelect = ({ selectedOption, setSelectedOption}: SubscriptionPlanSelectProps) => {
+  const subscriptionOptions = [
+    { value: MembershipSubscriptionPeriod.Monthly, displayName: 'Monthly - £12' },
+    { value: MembershipSubscriptionPeriod.Quarterly, displayName: '3 Months - £30' },
+    { value: MembershipSubscriptionPeriod.Annualy, displayName: '1 Year - £100' },
+  ]
+
+  return (
+    <Select
+      onChange={(value) => {setSelectedOption(value)}}
```

Comment:
```suggestion
      onChange={setSelectedOption(value)}
```

lets the caller `useCallback` if they want

Reply:
TypeScript explodes when doing this so I left it for the moment but we can definitely address it before merging!

Reply:
Never mind! ✅